lesson_1/task_4/Solution.java

```diff
@@ -0,0 +1,14 @@
+public class Solution {
+    public static void main (String [] args) {
+        int n = 4562;
+        String s = String.valueOf(n);
+        if(s.length()==4){
+            for(int i = s.length()-1; i>=0; i--){
```

Comment:
According to the code convention, you should type " " before and after arithmetic operators (i >= 0)

Reply:
done! thanks)